<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wizard</class>
 <widget class="ConditionalWizard" name="Wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard</string>
  </property>
  <widget class="QWizardPage" name="wizardPage_select_files">
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QComboBox" name="operation_comboBox">
      <item>
       <property name="text">
        <string>STM32CubeMX ==&gt; Autodesk Eagle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Autodesk Eagle ==&gt; STM32CubeMX</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="ResizeStackWidget" name="source_stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_11">
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>STM32CubeMX</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_6" native="true">
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Select Source:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="source_stm_lineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="source_stm_pushButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Autodesk Eagle</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_5" native="true">
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Select Schematic:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="source_eagle_sch_lineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="source_eagle_sch_pushButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Select IC:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="source_eagle_ic_lineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="source_eagle_ic_pushButton">
             <property name="text">
              <string>Select...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ResizeStackWidget" name="sink_stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_12">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>STM32CubeMX</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_7" native="true">
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Select source:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="sink_stm_lineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="sink_stm_pushButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Autodesk Eagle</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_8" native="true">
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Select schematic:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="sink_eagle_sch_lineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="sink_eagle_sch_pushButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_10" native="true">
          <layout class="QGridLayout" name="gridLayout_14">
           <item row="0" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Select board:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="sink_eagle_brd_lineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="sink_eagle_brd_pushButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_9" native="true">
          <layout class="QGridLayout" name="gridLayout_13">
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Select IC:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="sink_eagle_ic_lineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="sink_eagle_ic_pushButton">
             <property name="text">
              <string>Select...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_2">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Changelog:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Renamed nets from ... to ...:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="changelog_tableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Warnings and Errors:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QTableView" name="error_tableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>30</number>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Save result to:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="ResizeStackWidget" name="target_stackedWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_13">
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>STM32CubeMX</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_11" native="true">
          <layout class="QGridLayout" name="gridLayout_15">
           <item row="0" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Select target:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="target_stm_lineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="target_stm_pushButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Autodesk Eagle</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_12" native="true">
          <layout class="QGridLayout" name="gridLayout_16">
           <item row="0" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Select target schematic:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="target_eagle_sch_lineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="target_eagle_sch_pushButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_13" native="true">
          <layout class="QGridLayout" name="gridLayout_17">
           <item row="0" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Select target board:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="target_eagle_brd_lineEdit"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="target_eagle_brd_pushButton">
             <property name="text">
              <string>Browse...</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ResizeStackWidget</class>
   <extends>QStackedWidget</extends>
   <header>resizestackwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConditionalWizard</class>
   <extends>QWizard</extends>
   <header>conditionalwizard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
